cmd/panfs.go

```diff
@@ -428,6 +434,48 @@ func (fs *PANFSObjects) getBucketDir(ctx context.Context, bucket string) (string
 	return bucketDir, nil
 }
 
+// loadBucketMetadata loads bucket metadata. This function is slightly different from bucket-metadata:loadBucketMetadata
```

Comment:
Instead of mentioning that the function is different, maybe just mention that if the metadata is not found, an error is returned?

Reply:
I would not spend time for this function (at least for now) as it will be completely reworked to interacting with the configuration agent. At the moment that is just a temporary solution